package/MDAnalysis/core/universe.py

```diff
@@ -202,42 +207,32 @@ def _make_groupclasses(self):
         Topology.
```

Comment:
Change the method name to `_make_classes`, since it makes both `Group` and `Component` classes.